src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+@import '../button/borderless';
+
+$iui-color-field-size: $iui-baseline * 19;
+
+
+@mixin iui-color-picker {
+  @include iui-dropdown;
+  max-width: ($iui-l * 10) + ($iui-xxs * 10) + ($iui-s * 9) + ($iui-sm * 2);
+  // max-width: (10 swatches per row) + (border) + (gaps) + (container padding)
+  max-height: ($iui-l * 10) + ($iui-xxs * 10) + ($iui-s * 9) + ($iui-baseline * 2);
+  // max-height: (10 rows) + (border) + (gaps) + (container padding)
+  box-sizing: border-box;
+  border-radius: $iui-border-radius;
+  display: flex;
+  padding: $iui-baseline $iui-sm;
+  user-select: none;
+  overflow-y: overlay;
+
+  &.iui-advanced {
+    max-width: ($iui-l * 7) + ($iui-xxs * 7) + ($iui-s * 6) + ($iui-sm * 2) + ($iui-s);
+    flex-wrap: wrap;
+  }
+}
+
+@mixin iui-color-swatch {
+  @include iui-focus;
+  height: $iui-l;
+  width: $iui-l;
+  cursor: pointer;
+  border-radius: $iui-border-radius-5;
+  background-color: var(--swatch-color);
+  margin-bottom: $iui-xs;
+  margin-right: $iui-xs;
+
+  @supports (gap: $iui-s) {
+    margin-bottom: 0;
+    margin-right: 0;
+  }
+
+  @include themed {
+    box-shadow: inset 0 0 0 $iui-xxs
+      rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+    border: 1px solid t(iui-color-background-1);
+  }
+
+  &.iui-active {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs t(iui-color-foreground-primary);
+    }
+  }
+
+  &:hover {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5)),
+        inset 0 0 0 $iui-xxs
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
```

Comment:
This border around the color swatch was done intentionally to match the color picker from the ux standards 
![image](https://user-images.githubusercontent.com/88331924/131376104-442f676e-dbaa-45c4-a6c1-d919caf0521b.png)

But mines looks different because of the inset box shadow. This is what it look like without it, should I just remove it?
![image](https://user-images.githubusercontent.com/88331924/131376427-45d9ac27-0008-413d-92ff-d3f0c8e15c92.png)

Reply:
I would say remove it, but it still doesn't look great (definitely better than the gap though).

Reply:
Border was removed

Reply:
oh it looks especially terrible in dark theme now, even worse than before
![image](https://user-images.githubusercontent.com/9084735/131573021-be297164-3ad1-4f02-8fbb-4fd887495154.png)